### ISSUE TITLE
Scroll to mixer channel matching currently selected staff in score

### ISSUE THREAD

**Issue by wizofaus** [musescore/MuseScore#23662]
### Your idea

Would like the ability to quickly scroll to the mixer channel that matches the (first) selected staff in the score, or where the input cursor currently is.

### Problem to be solved

When working on large orchestral scores with 25+ channels it can be quite slow/painful to find the relevant mixer channel for the staff you're currently working on.  Could just be a button on the mixer panel to "scroll to current" or similar.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by prodoelmit** [comment]
I'll take this

How I see it: 
* Add option to enable this behavior in preferences in Audio/Midi -> Mixer 
* When score's selection changes, scroll (smoothly, with some kind of easing) to desired offset (if needed) 
* (I'm not sure how it will look, so I'll try to keep the code tweakable)

**Comment by prodoelmit** [comment]
@wizofaus please take a look: https://youtube.com/shorts/vvWseggGwbg?feature=share 

From my perspective - even after scrolling it still requires some time for eyes to find the channel we need. Maybe some animated or colored identification would make it easier

**Comment by avvvvve** [comment]
@prodoelmit Agreed, this should have an additional visual indicator as to which channel(s) are selected in the score. But thanks for the proof of concept so we could identify what needs designing!

We'll take this off of community projects for now and ready a design for our internal team to handle along with some other slated mixed improvements.

**Comment by cbjeukendrup** [comment]
The above sounds slightly related to https://github.com/musescore/MuseScore/issues/23302
